packages/core/test/discovery.test.js

```diff
@@ -2885,4 +2885,124 @@ describe('Discovery', () => {
       }));
     });
   });
+
+  describe('Handles multiple root resources', () => {
+    it('gathers resources for a snapshot', async () => {
+      let DOM1 = testDOM.replace('Percy!', 'Percy! at 370');
+      let DOM2 = testDOM.replace('Percy!', 'Percy! at 765');
+      const capturedResource = {
```

Comment:
Please update this test such that there is 1 resource in DOM1 which is not there in DOM2 or testDOM and similar for DOM2 

So that your assertions will contain those 2 unique resources as well.